ui/components/ui/button-group/README.mdx

```diff
@@ -0,0 +1,77 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import ButtonGroup from '.';
+
+# Button Group
+
+Button Group is a wrapper for buttons to align them horizontally
+
+<Canvas>
+  <Story id="ui-components-ui-button-group-button-group-stories-js--default-story" />
+</Canvas>
+
+## Component API
+
+<ArgsTable of={ButtonGroup} />
+
+## Usage
+
+The following describes the props and example usage for this component.
+
+| Prop Type                  | Description                                                                         |
+| -------------------------- | ----------------------------------------------------------------------------------- |
+| `defaultActiveButtonIndex` | change button active order                                                          |
+| `noButtonActiveByDefault`  | no buttons are active before clicked by the user                                     |
+| `disabled`                 | disable every button inside button group                                          |
+| `className`                | classname is used for children prop, changing it in storybook won't change anything |
+| `style`                    | add style for button group component                                             |
+| `newActiveButtonIndex`     | update value of active button in button group component                           |
+| `variant`                  | options for rendering types of buttons, consists of 'default' and 'radiogroup'         |
```

Comment:
This is awesome and I just realized if we just jsDocs formatting we can get the description to show up in the `ArgsTable` I'll create an issue to update the Button component and contribution docs to include this standard. This should encourage and help maintain good documentation practices. If you move all of your descriptions from here to `ui/components/ui/button-group/button-group.component.js`. You can remove the table and it's one less thing that will need to update if props change.

```
export default class ButtonGroup extends PureComponent {
  static propTypes = {
    /**
     * Change button active order
     */
    defaultActiveButtonIndex: PropTypes.number,
```
> Example of first prop description

How it looks in the `ArgsTable`

<img width="1437" alt="Screen Shot 2021-11-09 at 8 36 51 AM" src="https://user-images.githubusercontent.com/8112138/140966021-169746a3-f902-4964-b92a-2076cb1d43f1.png">